platform/opendingux/inputmap.c

```diff
@@ -137,8 +140,10 @@ void plat_target_setup_input(void)
 	if (strcmp(plat_device, "miyoo") == 0) {
 		/* swapped A/B and X/Y keys */
 		keyswap(SDLK_LALT, SDLK_LCTRL);
+		keyswap(SDLK_MENU, SDLK_ESCAPE);
```

Comment:
not working with current setup...so dumping that idea, also missed the point of nameswap() obviously.